src/sagemaker/workflow/steps.py

```diff
@@ -93,6 +93,26 @@ def ref(self) -> Dict[str, str]:
         return {"Name": self.name}
 
 
+@attr.s
+class CacheConfig:
+    """Step to cache pipeline workflow.
+
+    Attributes:
+        enable_caching (bool): To enable step caching. Off by default.
```

Comment:
I would give the value `False` rather than "Off" and follow the docstring convention found elsewhere to indicate what the default value is.